web-client/components/Button.js

```diff
@@ -1,11 +1,24 @@
-function Button(props) {
+import PropTypes from 'prop-types';
+
+const Button = ({ buttonName, onClick }) => {
     return (
-        <button className= "transition duration-700 ease-in-out bg-[#9945FF] hover:bg-gradient-to-tr hover:from-[#9945FF] hover:to-[#14F195] text-white px-6 py-3 font-bold rounded" onClick={props.onClick}>
-            {props.buttonName}
+        <button className= "transition duration-700 ease-in-out bg-[#9945FF] hover:bg-gradient-to-tr hover:from-[#9945FF] hover:to-[#14F195] text-white px-6 py-3 font-bold rounded" onClick={onClick}>
+            {buttonName}
         </button>
     );
+
 }
 
 export default Button;
 
+Button.propTypes = {
```

Comment:
Added propTypes in order to get a bit of type checking. Will give errors when developing if the wrong type is passed in.

Reply:
fancy.